distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1522,11 +1516,11 @@ type Table
          retyped to the `Mixed` type to indicate that intention. Note that the
          `Mixed` type may not be supported by most Database backends.
     union : (Table | Vector Table) -> Match_Columns -> Boolean | Report_Unmatched -> Boolean -> Problem_Behavior -> Table
-    union self tables match_columns=Match_Columns.By_Name keep_unmatched_columns=Report_Unmatched allow_type_widening=True on_problems=Report_Warning =
+    union self tables:Vector|Table match_columns=Match_Columns.By_Name keep_unmatched_columns=Report_Unmatched allow_type_widening=True on_problems=Report_Warning =
         all_tables = case tables of
-            v : Vector -> [self] + v
-            single_table -> [self, single_table]
-        all_tables.all (check_db_table "tables") . if_not_error <|
+            v : Vector -> [self] + (v.map t-> Table.from t)
+            single_table -> [self, Table.from single_table]
```

Comment:
I will remove the `Table.from single_table` then.

But the one within the `map` is still needed, because we check only `v : Vector` and not `v : Vector Table`.